src/pages/sa-trx/[functionId]/forms/SaleTransactionForm.js

```diff
@@ -577,6 +605,13 @@ export default function SaleTransactionForm({ labels, access, recordId, function
       name: 'unitPrice',
       updateOn: 'blur',
       async onChange({ row: { update, newRow } }) {
+        const ItemConvertPrice = await getItemConvertPrice(newRow.itemId)
+        const unitPrice = parseFloat(newRow.unitPrice || 0).toFixed(3)
+        const minPrice = parseFloat(ItemConvertPrice?.minPrice || 0).toFixed(3)
```

Comment:
why u re-requesting getICP, on sku selection u already request it and now minPrice is stored in newRow